MapboxAndroidSDK/src/main/java/com/mapbox/mapboxsdk/customsettings/Settings.java

```diff
@@ -0,0 +1,39 @@
+package com.mapbox.mapboxsdk.customsettings;
```

Comment:
I don't think we want this in the Mapbox Android SDK. Work done within the SDK should be something that would make sense to be in the SDK. We send in pull requests occasionally to Mapbox, and this feature is specific to OpenMapKit